src/components/ThumbnailImage.js

```diff
@@ -54,8 +72,12 @@ class ThumbnailImage extends PureComponent {
         } else {
             thumbnailScreenHeight = Math.round(thumbnailScreenWidth * aspectRatio);
         }
+        return {thumbnailWidth: thumbnailScreenWidth, thumbnailHeight: Math.max(40, thumbnailScreenHeight)};
+    }
 
-        this.setState({thumbnailWidth: thumbnailScreenWidth, thumbnailHeight: Math.max(40, thumbnailScreenHeight)});
+    updateImageSize({width, height}) {
```

Comment:
missing js docs.

Reply:
Apparently it wasn't there before either but can you add them.

Reply:
Fixed.